library/src/main/scala/scala/scalajs/runtime/Bits.scala

```diff
@@ -16,9 +16,18 @@ import js.typedarray
 /** Low-level stuff. */
 object Bits {
 
-  val areTypedArraysSupported = js.DynamicImplicits.truthValue(
-      global.ArrayBuffer && global.Int32Array &&
-      global.Float32Array && global.Float64Array)
+  private[this] val _areTypedArraysSupported = {
```

Comment:
Er ... there is one (two actually, one in the val and one in the def). What is not clear?